src/main/java/seedu/address/model/module/Module.java

```diff
@@ -30,45 +31,54 @@ public class Module {
     private final ModuleTitle moduleTitle;
     private final TaskList tasks;
     private final Set<Link> links = new HashSet<>();
```

Comment:
```suggestion
    private final Set<Link> links;
```
Hmm was there an error in resolving the merge conflict? I believe it was previously agreed that `links`, `persons`  and `tasks` will all be initialised as `null`.